### ISSUE TITLE
batchnorm layer inconsistency

### ISSUE THREAD

**Issue by ShervinAr** [NVIDIA/DIGITS#1648]
I am trying to load a pre-trained caffe model in Digits but get the following error message:

_ERROR: Check failed: target_blobs.size() == source_layer.blobs_size() (5 vs. 3) Incompatible number of blobs for layer conv1/bn_

the error is related to a batch normalization layer where in Digits (nvCaffe) it expects five blobs whereas in pure Caffe (where the pre-trained model is trained) it has three blobs!
any comments how to work around this is greatly appreciated.

**Comment by TheMikeyR** [comment]
I have the same issue, but mine is a different amount of blob required. 

_ERROR: Check failed: target_blobs.size() == source_layer.blobs_size() (2 vs. 1) Incompatible number of blobs for layer conv1/7x7_s2_

**Comment by CEDAclaire** [comment]
Are you using MobileNet?

**Comment by CEDAclaire** [comment]
@TheMikeyR Your issue can can probably be fixed by changing the convolution bias term from false to true. This will give you the second blob that you need. 
@ShervinAr Are you using MobileNet?  In MobileNet doing that will just move the error from conv1/bn to conv1. I am working through the MobileNet prototxt errors too. A MobileNet creator addressed this problem at 

https://github.com/shicai/MobileNet-Caffe/issues/6
